packages/main/src/themes/RatingIndicator.css

```diff
@@ -0,0 +1,37 @@
+:host(:not([hidden])) {
+	display: inline-block;
+	font-size: 1.5rem;
+	cursor: pointer;
+}
+
+:host([disabled]),
+:host([read-only]) {
```

Comment:
It is strange the disabled and read-only states to share the same opacity, is this really by design?